pkg/minikube/kubernetes_versions/kubernetes_versions.go

```diff
@@ -61,13 +61,40 @@ func getJson(url string, target *K8sReleases) error {
 	return json.NewDecoder(r.Body).Decode(target)
 }
 
+var cachedK8sVersions = make(K8sReleases, 0)
+
 func GetK8sVersionsFromURL(url string) (K8sReleases, error) {
+	if len(cachedK8sVersions) != 0 {
+		glog.Infof("Using cached localkube versions: %v", cachedK8sVersions)
```

Comment:
Nope, I figured I would add it to help debug any future issues that might quietly happen as a result of the caching.  Should be fine to remove though.